src/v/config/tests/enum_property_test.cc

```diff
@@ -76,10 +76,10 @@ SEASTAR_THREAD_TEST_CASE(enum_property_values) {
 
     auto opt_values = cfg.opt_enum_str.enum_values();
     BOOST_CHECK_EQUAL(opt_values.size(), 4);
-    BOOST_CHECK_EQUAL(opt_values[0], "{nullopt}");
-    BOOST_CHECK_EQUAL(opt_values[1], "{foo}");
-    BOOST_CHECK_EQUAL(opt_values[2], "{bar}");
-    BOOST_CHECK_EQUAL(opt_values[3], "{baz}");
+    BOOST_CHECK_EQUAL(opt_values[0], "null");
+    BOOST_CHECK_EQUAL(opt_values[1], "foo");
```

Comment:
>fix enum values formatting

This says that we are fixing it, but it isn't obvious why one formatting is correct or wrong. can you add context to the commit message?

Reply:
Done. PTAL. https://github.com/redpanda-data/redpanda/pull/24919/commits/12818cec2281a6843a5d9ec0a40da70ef297b706